lib/OpenQA/WebSockets/Server.pm

```diff
@@ -21,7 +21,7 @@ use Try::Tiny;
 use OpenQA::IPC;
 use OpenQA::Utils qw(log_debug log_warning log_error);
 use OpenQA::Schema;
-use OpenQA::ServerStartup;
+use OpenQA::Base;
```

Comment:
if you do something like: `use base OpenQA::Base`, then later on (@ line 459) you could do: `read_config(app)` and would look much cleaner